### PR TITLE
added text clean demo

### PR DESCRIPTION
**Pull request by allan8392**
Addresses #45

### CODE REVIEW

File: docs/example.ipynb [hunk 1]
```diff
@@ -72,15 +72,20 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**one or two sentences describing what this function does in the context of usage and functionality**"
+    "`text_clean()` cleans raw text for further text processing. This function will lower case all characters, delete punctuations as well as numbers and split words by spaces. All functions in this package will call `text_clean()` before transforming the text to other representations. The user can use this function to clean texts before feeding the text to their own algorithm.\n",
```
Comment (jy1909) [inline docs/example.ipynb]: Small change for better narrative.

```suggestion
    "`text_clean()` cleans raw text for further text processing. This function will convert all characters to lower case, remove punctuations as well as numbers, and split words by spaces. All other functions in this package will call `text_clean()` before transforming the text to other representations, and therefore accept raw text as input. The user can also use this function to clean texts before feeding the texts to another algorithm of their choice.\n",
```

**Review by jy1909 — COMMENTED** [review]
Great work! One small change for better narrative.

**Review by MoNorouzi23 — APPROVED** [review]

**Comment by MoNorouzi23** [comment]
it was great

**Review by jy1909 — APPROVED** [review]

**Comment by jy1909** [comment]
I just approved it as well. The merge conflict still needs to be resolved before we can merge.

**Pull request merged**